labelCloud/model/point_cloud.py

```diff
@@ -323,6 +323,25 @@ def reset_perspective(self) -> None:
         self.trans_x, self.trans_y, self.trans_z = self.init_rotation
         self.rot_x, self.rot_y, self.rot_z = self.init_rotation
 
+    def get_filtered_pointcloud(
+        self, indicies: npt.NDArray[np.bool_]
+    ) -> Optional["PointCloud"]:
+        assert self.points is not None
+        assert self.colors is not None
+        points = self.points[indicies]
+        if points.shape[0] > 0:
+            colors = self.colors[indicies]
+            labels = self.labels[indicies] if self.labels is not None else None
+            path = self.path.parent / (self.path.stem + "_cropped" + self.path.suffix)
+            return PointCloud(
+                path=path,
+                points=points,
+                colors=colors,
+                segmentation_labels=labels,
+                write_buffer=False,
+            )
+        return None
```

Comment:
Style detail: Usually the happy path is at the end of a method:


```python
...

if points.shape[0] == 0:
  return None

...

return pointcloud
```

Reply:
changed here [65f1363](https://github.com/ch-sa/labelCloud/pull/129/commits/65f1363624fca6cbf122ed0291bfa71376272ea6)